target/linux/ath79/image/generic.mk

```diff
@@ -75,6 +75,14 @@ define Device/seama
   SEAMA_SIGNATURE :=
 endef
 
+define Device/8dev_carambola2
+  ATH_SOC := ar9331
+  DEVICE_TITLE := 8devices Carambola2
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-chipidea2
+  IMAGE_SIZE := 16000k
```

Comment:
FYI, it's a matter of personal preference and you can leave it as it is, but `16m` would work as well and might be more readable.

Reply:
@ynezz 
IMAGE_SIZE should match the size of firmware partition, which in this case it's 0xfa0000=16384000=16000k.
 16m=16384k

Reply:
> 0xfa0000=16384000=16000k.
> 16m=16384k

Yes, correct. Another late night/early morning hickup :( Sorry for the noise.